R/fotmob_stats.R

```diff
@@ -101,7 +101,7 @@
   see_all_button <- page %>% rvest::html_elements(".SeeAllButton")
   has_see_all_button <- length(see_all_button) > 0
 
-  options <- page |> rvest::html_elements("option")
+  options <- page %>% rvest::html_elements("option")
```

Comment:
whoops! since we're not requiring that users have R 4.1, i should still use magrittr's pipe everywhere

Reply:
Oh yeah I've been meaning to bring this up! Thanks for catching it!